containers/daap-python-base/tests/unit/data_platform_paths_test.py

```diff
@@ -140,11 +180,15 @@ def test_extraction_config():
 
     config = DataProductConfig(
         name="my-database",
-        table_name="some-table",
-        bucket_name="a-bucket",
+        raw_data_bucket="a-bucket",
+        curated_data_bucket="a-bucket",
+        landing_zone_bucket="a-bucket",
+        metadata_bucket="a-bucket",
     )
 
-    extraction = config.extraction_config(uuid_value=uuid_value, timestamp=timestamp)
+    element = config.element("some-table")
+
+    extraction = element.extraction_config(uuid_value=uuid_value, timestamp=timestamp)
```

Comment:
```suggestion
    extraction = element.extraction_instance(uuid_value=uuid_value, timestamp=timestamp)
```
If changing [method name](https://github.com/ministryofjustice/data-platform/pull/1549/files#diff-f47b07dfcbf000013d2c9d5ec4009c2b492803a64344d8da662490171a6184dcR182), needs changing here too